gradio/chat_interface.py

```diff
@@ -130,6 +131,7 @@ def __init__(
             show_progress: how to show the progress animation while event is running: "full" shows a spinner which covers the output component area as well as a runtime display in the upper right corner, "minimal" only shows the runtime display, "hidden" shows no progress animation at all
             fill_height: if True, the chat interface will expand to the height of window.
             fill_width: Whether to horizontally expand to fill container fully. If False, centers and constrains app to a maximum width.
+            api_name: the name of the API endpoint to use for the chat interface. Defaults to "/chat". Set to False to disable the API endpoint.
```

Comment:
Nit: so when a user specified the `api_name`, they don't add the leading slash:
```suggestion
            api_name: the name of the API endpoint to use for the chat interface. Defaults to "chat". Set to False to disable the API endpoint.
```